zone/raids.cpp

```diff
@@ -1164,6 +1254,11 @@ void Raid::QueuePacket(const EQApplicationPacket *app, bool ack_req)
 
 void Raid::SendMakeLeaderPacket(const char *who) //30
 {
+
+	if (RuleB(Bots, Enabled) && entity_list.GetBotByBotName(who) && members[GetPlayerIndex(who)].IsBot) {
```

Comment:
This seems crash prone, we should check that player index is valid and not out of bounds

Reply:
it actually appears to be redundant, since ```entity_list.GetBotByBotName(who) ``` would return a Bot* if **who** is a Bot. I'll clean this up.